velox/docs/configs.rst

```diff
@@ -206,6 +211,12 @@ Spilling
        If the limit is zero, then the spiller always spills a previously spilled partition if it has any data. This is
        to avoid spill from a partition with a small amount of data which might result in generating too many small
        spilled files.
+   * - spill_compression_codec
+     - string
+     - none
+     - Specifies the compression algorithm type to compress the spilled data before write to disk to trade CPU for IO
+       efficiency. The supported compression codecs are: ZLIB, SNAPPY, LZO, ZSTD, LZ4 and GZIP.
```

Comment:
Is this value case sensitive or not? Can I specify NONE or do I have to specify none? Will nOnE work?

Reply:
It is case sensitive but we could make it non-case sensitive in followup.